ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2627,6 +2627,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8
+- as: ibmcloud-ipi-proxy-private-p2-f14
```

Comment:
Based on this new job, `ibmcloud-ipi-private-fips-p2-f7` can be adjust to `f14`.

Reply:
Even `ibmcloud-ipi-p2-f7` can be dropped now, because it is being covered by `ibmcloud-ipi-workers-rhel8-p2-f14`.